nbs/01_datasets.ipynb

```diff
@@ -252,7 +252,8 @@
     "            if self.shuffle:\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #50.                batch_data = list(map(tuple, zip(*batch_data)))
</pre><p>delete this line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BirkhoffG/cfnet/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>